dotCMS/src/integration-test/java/com/dotmarketing/portlets/categories/business/CategoryAPITest.java

```diff
@@ -1084,4 +1086,150 @@ private ContentType createContentTypeWithCatAndTextField(final Category parentCa
 
         return APILocator.getContentTypeAPI(user).find(type.inode());
     }
+
+    /**
+     * Method to test: {@link CategoryAPI#isParent(Category, Category, User, boolean)}
+     * Given scenario: Create a Category with 3 levels of depth:
+     *          Parent Category
+     *                  Child Category
+     *                          Grand Child Category
+     *
+     *                  And check if the grand child category belongs to the parent category
+     * Expected result: true, since the grand child category is a sub sub category
+     */
+    @Test
+    public void test_isParent_givenGrandChildCategoryBelongsToParentCategory_returnTrue()
+            throws DotSecurityException, DotDataException {
+        final List<Category> categoriesToDelete = Lists.newArrayList();
+        final CategoryAPI categoryAPI = APILocator.getCategoryAPI();
+        try {
+            //Create Parent Category.
+            final Category parentCategory = new CategoryDataGen()
+                    .setCategoryName("CT-Category-Parent")
+                    .setKey("parent")
+                    .setCategoryVelocityVarName("parent")
+                    .nextPersisted();
+
+            categoriesToDelete.add(parentCategory);
+
+            //Create First Child Category.
+            final Category childCategoryA = new CategoryDataGen()
+                    .setCategoryName("CT-Category-A")
+                    .setKey("categoryA")
+                    .setCategoryVelocityVarName("categoryA")
+                    .next();
+
+            categoriesToDelete.add(childCategoryA);
+
+            //Second Level Category.
+            Category childCategoryA_1 = new CategoryDataGen()
+                    .setCategoryName("CT-Category-A-1")
+                    .setKey("categoryA-1")
+                    .setCategoryVelocityVarName("categoryA-1")
+                    .next();
+
+            categoriesToDelete.add(childCategoryA_1);
+
+            categoryAPI.save(parentCategory, childCategoryA, user, false);
+            categoryAPI.save(childCategoryA, childCategoryA_1, user, false);
+
+            Assert.assertTrue(categoryAPI.isParent(childCategoryA_1,parentCategory,user,true));
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Assert.assertTrue' due to existing static import 'org.junit.Assert.assertTrue'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6106194)